python/mxnet/symbol/symbol.py

```diff
@@ -2665,11 +2665,30 @@ def _optimize_for_dynamic_shape_op(self, is_np_array):
             - If true, output type is np symbol, otherwise nd symbol.
         """
         out = SymbolHandle()
-        check_call(_LIB.MXOptimizeForDynamicShapeOp(self.handle, ctypes.byref(out)))
+        has_dynamic_shape = ctypes.c_bool(False)
+
+        # Pass static_allco and static_shape flags into c_api
+        key_list = []
+        val_list = []
+        for flag in flags:
+            key, val = flag[0], flag[1]
+            if key == ['static_alloc', 'static_shape']:
```

Comment:
shouldnt it be `if key in ['static_alloc', 'static_shape']:`

Reply:
Yes, thanks for catching that